fastlane/Fastfile

```diff
@@ -537,6 +537,45 @@ platform :ios do
     end
   end
 
+  # This lane publishes a release on GitHub, tagging it on git, and backmerges the current release branch into the next release/ branch
+  #
+  # @param [Boolean] skip_confirm (default: false) If set, will skip the confirmation prompt before running the rest of the lane
+  #
+  # @example Running the lane
+  #          bundle exec fastlane publish_release_and_cleanup skip_confirm:true
+  #
+  lane :publish_release_and_cleanup do |skip_confirm: false|
```

Comment:
Same 💭  as https://github.com/woocommerce/woocommerce-android/pull/12450#discussion_r1739151450

Reply:
I've updated the PR after the release of [release-toolkit 12.0.0](https://rubygems.org/gems/fastlane-plugin-wpmreleasetoolkit/versions/12.0.0).